### PR TITLE
5 get article id comments

### PR DESCRIPTION
**Pull request by katmiah**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -114,4 +122,45 @@ describe("GET /api/articles", () => {
...
   })
+})
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article ID", () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({ body }) => {
+      console.log(body, "first log")
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:133]: Please remove and don't commit debugging logs in a PR next time.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -114,4 +122,45 @@ describe("GET /api/articles", () => {
...
+      const comments = body.comments
+      console.log(comments, "second log")
+      comments.forEach((comment) => {
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:136]: Length of the array must be checked to avoid false positive tests.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -114,4 +122,45 @@ describe("GET /api/articles", () => {
...
+  })
+
+  test("400: Responds with error message if article ID is invalid", () => {
+    return request(app)
+    .get("/api/articles/banana/comments")
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.message).toBe("Bad request.")
+    })
+  })
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:165]: Please include a test with status 200 and an empty array where there are no comments associated with an article. 

You will need to check the reason for the query returning empty rows:
article exists but has no comments.
article does not exist.

**Review by nc-bela-bertalan — COMMENTED** [review]
Hi Katie, please see my comments and add the missing test.
